portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random fact.
```

Comment:
Maybe add a random fact to ...?
or show a random fact on ...?

Reply:
Fixed